sdks/swift/Mentat/Mentat/Extensions/UnsafeMutablePointerResult+Unwrap.swift

```diff
@@ -11,37 +11,43 @@
 import Foundation
 import MentatStore
 
-public extension Result {
+extension UnsafeMutablePointer where Pointee == Result {
     /**
-     Force unwraps a result.
-     Expects there to be a value attached and throws an error is there is not.
-
+     Unwraps an optional result, yielding either a successful value or a nil.
```

Comment:
nit: Just "or nil".  I don't think there are multiple `nil` instances, even in Swift.